enterprise/javaee.wildfly/src/org/netbeans/modules/javaee/wildfly/util/WildFlyProperties.java

```diff
@@ -232,12 +232,16 @@ private List<URL> selectJars(FileObject file) {
     }
 
     public List<URL> getClasses() {
-        List<URL> list = selectJars(FileUtil.toFileObject(new File(getModulePath("javax"))));
+        List<URL> classes = new ArrayList<>();
+
+        classes.addAll(selectJars(FileUtil.toFileObject(new File(getModulePath("jakarta")))));
+        classes.addAll(selectJars(FileUtil.toFileObject(new File(getModulePath("javax")))));
```

Comment:
i suppose its ok to add both since the namespace doesn't conflict?

Reply:
Afaik there's no release providing both namespaces and if the module is not found it's not added (that's what happened before when you add a new wildfly with jakarta namespace) - So the only thing I can imagine this might cause problems is when both apis are provided by wildfly/jboss eap. I slightly remember that there was some time where both namespaces were supported, but I think I remember that was done via some custom `ClassLoader`.

Reply:
wildfly seems to provide both: 
```
/wildfly-29.0.0.Beta1/modules/system/layers/base/jakarta/
/wildfly-29.0.0.Beta1/modules/system/layers/base/javax/
```
same for /wildfly-28.0.0.Final and /wildfly-27.0.1.Final

Reply:
The folder might be present, but the `javax` folder does not contain the api `jar` files:

```console
find javax -name '*.jar'
javax/orb/api/main/openjdk-orb-9.0.2.Final.jar
javax/wsdl4j/api/main/wsdl4j-1.6.3.jar
```

```console
❯ find jakarta -name '*.jar'
jakarta/interceptor/api/main/jakarta.interceptor-api-2.1.0.jar
jakarta/security/auth/message/api/main/jakarta.authentication-api-3.0.0.jar
jakarta/security/jacc/api/main/jakarta.authorization-api-2.1.0.jar
jakarta/security/enterprise/api/main/jakarta.security.enterprise-api-3.0.0.jar
jakarta/json/api/main/jakarta.json-api-2.1.2.jar
jakarta/json/bind/api/main/jakarta.json.bind-api-3.0.0.jar
jakarta/inject/api/main/jakarta.inject-api-2.0.1.jar
jakarta/enterprise/api/main/jakarta.enterprise.cdi-api-4.0.1.jar
jakarta/enterprise/api/main/jakarta.enterprise.lang-model-4.0.1.jar
jakarta/enterprise/concurrent/api/main/jakarta.enterprise.concurrent-api-3.0.2.jar
jakarta/websocket/api/main/jakarta.websocket-api-2.1.0-jbossorg-2.jar
jakarta/websocket/api/main/jakarta.websocket-client-api-2.1.0-jbossorg-2.jar
jakarta/servlet/jstl/api/main/jakarta.servlet.jsp.jstl-api-3.0.0.jar
jakarta/servlet/jstl/api/main/jakarta.servlet.jsp.jstl-3.0.0.jar
jakarta/servlet/jsp/api/main/jakarta.servlet.jsp-api-3.1.0.jar
jakarta/servlet/api/main/jakarta.servlet-api-6.0.0.jar
jakarta/el/api/main/jboss-el-api_5.0_spec-4.0.1.Final.jar
jakarta/annotation/api/main/jakarta.annotation-api-2.1.1.jar
jakarta/transaction/api/main/jakarta.transaction-api-2.0.1.jar
jakarta/resource/api/main/jakarta.resource-api-2.1.0.jar
jakarta/activation/api/main/jakarta.activation-api-2.1.2.jar
jakarta/xml/bind/api/main/jakarta.xml.bind-api-4.0.0.jar
jakarta/xml/soap/api/main/jboss-saaj-api_3.0_spec-1.0.0.Final.jar
jakarta/xml/ws/api/main/jboss-jakarta-xml-ws-api_4.0_spec-1.0.0.Final.jar
jakarta/ejb/api/main/jakarta.ejb-api-4.0.1.jar
jakarta/persistence/api/main/jakarta.persistence-api-3.1.0.jar
jakarta/validation/api/main/jakarta.validation-api-3.0.2.jar
jakarta/ws/rs/api/main/jakarta.ws.rs-api-3.1.0.jar
jakarta/mail/api/main/jakarta.mail-api-2.1.2.jar
jakarta/jms/api/main/jakarta.jms-api-3.1.0.jar
jakarta/faces/impl/main/jakarta.faces-4.0.2.jar
jakarta/batch/api/main/jakarta.batch-api-2.1.1.jar
```

Reply:
you are right - interesting. The javax folder has only xmls in it.